src/main/java/org/zalando/pazuzu/feature/FeaturesResource.java

```diff
@@ -32,10 +32,16 @@ public FeaturesResource(FeatureService featureService) {
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<FeatureDto> listFeatures(
             @RequestParam(required = false, name = "name") String[] featureNames,
-            @RequestParam(required = false, name = "sorted") Integer sorting)
+            @RequestParam(required = false, name = "sorted") Integer sorting,
```

Comment:
Is your point that for some use cases you simple load features without transitive deps and the other way around for other use cases? I totally agree. But I don't like the idea of boolean parameter - looks like a hack-ish workaround. Maybe you find a better way of solving this.

I would do this in another PR, so we can merge this one.